LoggingKata.Test/TacoParserTests.cs

```diff
@@ -1,29 +1,64 @@
 using System;
+using System.Collections.Generic;
 using Xunit;
 
 namespace LoggingKata.Test
 {
     public class TacoParserTests
-    {
+    {
+
+
         [Fact]
         public void ShouldDoSomething()
         {
             // TODO: Complete Something, if anything
         }
 
         [Theory]
-        [InlineData("Example")]
+        [InlineData("")]
+        [InlineData("8,6,TacoBell")]
+        [InlineData("8,0,tacoBell")]
+        [InlineData("75356,879345,cheese")]
+        [InlineData("5.5,6.6,fred")]
         public void ShouldParse(string str)
         {
-            // TODO: Complete Should Parse
+            // arrange
+            TacoParser tacoParser = new TacoParser();
+
+

```

Comment:
These extra empty lines could be removed.  If space is needed, try to stick to a single empty line